sign/schnorr.go

```diff
@@ -0,0 +1,95 @@
+package sign
+
+import (
+	"bytes"
+	"crypto/sha256"
+	"errors"
+	"fmt"
+
+	"github.com/dedis/crypto/abstract"
+	"github.com/dedis/crypto/random"
+)
+
+// HashFunc is the hash function used to hash the message for the signature
+// generation and verification
+var HashFunc = sha256.New
```

Comment:
style: global variable that is used only once - can go into the local `hash`-function, IMO.